app/services/katello/pulp3/repository/yum.rb

```diff
@@ -266,42 +276,45 @@ def copy_content_from_mapping(repo_id_map, options = {})
           multi_copy_units(repo_id_map, dependency_solving)
         end
 
+        # rubocop:disable Metrics/MethodLength
         def copy_content_for_source(source_repository, options = {})
-          filters = [ContentViewErratumFilter, ContentViewPackageGroupFilter, ContentViewPackageFilter].collect do |filter_class|
-            filter_class.where(:id => options[:filter_ids])
-          end
-
-          filters.flatten!.compact!
+          if options.fetch(:import_filter, false)
+            copy_all(source_repository)
```

Comment:
i'm not a huge fan of the name 'import_filter', i'd rather describe what its doing.  Maybe 'copy_all' ?

Reply:
also, since 'copy_content_for_source' supports things that aren't supported for copy_all, would it make sense just to call copy all from the action?  (and maybe have a different action for copy_all vs copy_content_for_source)

Reply:
There is value in reusing copy content action and that action redirects to this class. I ve decided to now move the logic to the action and have that one call copy_all.